### PR TITLE
Skip chunks - they will be scanned on the final move anyway

### PR DESCRIPTION
**Pull request by VicDeo**
Ref https://github.com/owncloud/enterprise/issues/2227

### CODE REVIEW

File: lib/avirwrapper.php [hunk 1]
```diff
@@ -67,6 +67,7 @@ public function fopen($path, $mode){
 		if (is_resource($stream)
 			&& $this->isWritingMode($mode)
 			&& $this->isScannableSize(basename($path))
+			&& strpos($path, 'uploads/') !== 0
```
Comment (PVince81) [inline lib/avirwrapper.php]: can I trick this by having a path "myfolder/something/uploads/infected.exe" ?

Comment (PVince81) [reply lib/avirwrapper.php]: or is this check very subtle and means: only go there either when "uploads" is not at the start or when "uploads" is not found at all ? A PHP comment would be appreciated to clarify this implicit confusing logic

Comment (VicDeo) [reply lib/avirwrapper.php]: @PVince81 I think `strpos($path, 'uploads/') !== 0` is self-descriptive as it reads as `path does not start with`

**Comment by PVince81** [comment]
please backport to stable10

**Comment by VicDeo** [comment]
Stable10: https://github.com/owncloud/files_antivirus/pull/176

**Comment by PVince81** [comment]
Oracle setup fail, please investigate separately: https://travis-ci.org/owncloud/files_antivirus/jobs/269767617

**Comment by VicDeo** [comment]
seems to be a hiccup:
>Could not download from "http://pecl.php.net/get/oci8-2.1.7.tgz", cannot download "pecl/oci8" (Connection to `pecl.php.net:80' failed: Connection timed out)

**Pull request merged**